### DISCUSSION TITLE
Reset to defaults?

### DISCUSSION THREAD

**Discussion by swooped** [Lailloken/Exile-UI#414] [General]
Quick question I have used in the past and I'm trying to setup again when I initially loaded it showed the settings screen however now the settings button in the top left is not showing. 

Question is where does  Lailloken-UI store its user config files? I just want to start fresh if that makes sense and it seems like the files are saved outside the Lailloken-UI Directory somewhere.

**Comment by Lailloken** [comment]
There should be an `ini` folder, and inside there's a `config.ini` for global settings, as well as individual files for the other features.

**Comment by swooped** [comment]
lol that worked thank you appreciate everything!
